documentation/modules/creating-network-mapping.adoc

```diff
@@ -26,6 +26,7 @@ You can create one or more network mappings by using the {ocp} web console to ma
 The *Source networks* and *Target namespaces/networks* text boxes become active.
 
 . Select a source network and a target namespace/network from the list.
+. Optional: To save your mapping as a template, select *Save current mapping as a template* and enter a name for your mapping.
```

Comment:
Should this `Save current mapping as a template` option be added here? Is not on the `Create NetworkMap` window of `Migration -> NetworkMaps for virtualization`.

Reply:
yeah, I believe @qiyuann is right - every mapping that is created from the `NetworkMaps for virtualization` page produced a "mapping template". this option is supposed to appear only in the plan creation page/wizard